source/renderer/app/components/staking/stake-pools/StakePoolsTable.tsx

```diff
@@ -1,134 +1,34 @@
-  },
-  tableHeaderRoi: {
-    id: 'staking.stakePools.tableHeader.roi',
-    defaultMessage: '!!!Roi',
-    description: 'Table header "Roi" label on stake pools list view page',
-  },
-  tableHeaderCost: {
-    id: 'staking.stakePools.tableHeader.cost',
-    defaultMessage: '!!!Cost (ADA)',
-    description: 'Table header "Cost" label on stake pools list view page',
-  },
-  tableHeaderCostTooltip: {
-    id: 'staking.stakePools.tooltip.costPerEpochTooltip',
-    defaultMessage:
-      '!!!Fixed operational costs that the stake pool retains from any rewards earned during each epoch.',
-    description: '"Cost per epoch" tooltip for the Stake Pools Table.',
-  },
-  tableHeaderProducedBlocks: {
-    id: 'staking.stakePools.tableHeader.producedBlocks',
-    defaultMessage: '!!!Produced Blocks',
-    description:
-      'Table header "Produced Blocks" label on stake pools list view page',
-  },
-  tableHeaderProducedBlocksTooltip: {
-    id: 'staking.stakePools.tooltip.producedBlocksTooltip',
-    defaultMessage:
-      '!!!The total number of blocks the stake pool has produced.',
-    description: '"Blocks" tooltip for the Stake Pools Table.',
-  },
-  tableHeaderPotentialRewards: {
-    id: 'staking.stakePools.tableHeader.potentialRewards',
-    defaultMessage: '!!!Potential rewards',
-    description:
-      'Table header "Potential rewards" label on stake pools list view page',
-  },
-  tableHeaderPotentialRewardsTooltip: {
-    id: 'staking.stakePools.tooltip.potentialRewardsTooltip',
-    defaultMessage:
-      "!!!An estimation of the potential rewards you will earn per epoch if you delegate the intended amount of stake. The system looks at the pool's parameters and historical performance data to calculate potential rewards, assuming that the pool reaches optimal saturation.",
-    description: '"Rewards" tooltip for the Stake Pools Table.',
-  },
-  tableHeaderPledge: {
-    id: 'staking.stakePools.tableHeader.pledge',
-    defaultMessage: '!!!Pledge (ADA)',
-    description: 'Table header "Pledge" label on stake pools list view page',
-  },
-  tableHeaderPledgeTooltip: {
-    id: 'staking.stakePools.tooltip.pledgeTooltip',
-    defaultMessage:
-      '!!!The amount of stake that a pool operator contributes to a pool. Pools with higher pledge amounts earn more rewards for themselves and their delegators. Pools that do not honor their pledge earn zero rewards and accrue low ranking.',
-    description: '"Pledge" tooltip for the Stake Pools Table.',
-  },
-  tableHeaderRetiring: {
-    id: 'staking.stakePools.tableHeader.retiring',
-    defaultMessage: '!!!Retiring in',
-    description: 'Table header "Retiring" label on stake pools list view page',
-  },
-});
 export const defaultTableOrdering = {
-  ranking: 'asc',
-  ticker: 'asc',
-  saturation: 'asc',
-  cost: 'asc',
-  profitMargin: 'asc',
-  producedBlocks: 'desc',
-  nonMyopicMemberRewards: 'desc',
-  pledge: 'asc',
-  retiring: 'asc',
+  ranking: StakePoolsOrder.Asc,
```

Comment:
It would be good to increase type safety of this map by specifying type of its keys.
```
export const defaultTableOrdering: Record<keyof StakePool, StakePoolsOrder> = {
```